### ISSUE TITLE
Energy Impact goes down when closing the power_logger

### ISSUE THREAD

**Issue by ribalba** [green-coding-solutions/hog#14]
I've also noticed unexpected behavior: When I have the HOG.app open and then unload the power_logger, the Energy Impact decreases...
I suspect it's because the internal clock keeps running...? However, I would expect the Energy Impact to remain the same...? I don't know the formula, so it might not be a bug, but it's somehow unexpected.

**Comment by ribalba** [comment]
@ArneTR can you please retest with the new version https://github.com/green-coding-berlin/hog/releases/tag/0.3

**Comment by ArneTR** [comment]
Jap ,still happens.

To reproduce: 
- I started Docker Desktop on macOS
- Let it accumulate some energy impact. Energy impact rising constantly
- Close it
- Update date the *Last 5 Minutes list*
- Docker energy impact goes down

Screenshots of two minutes apart

<img width="1093" alt="Image" src="https://github.com/user-attachments/assets/12185006-b93d-495e-b480-651a1cb6ec03" />

<img width="1087" alt="Image" src="https://github.com/user-attachments/assets/42e288d9-b4cd-4b27-93cf-79e5d1840325" />

**Comment by ribalba** [comment]
Yes this makes sense as docker has used less energy in the last 5 minutes. As it hasn't used energy since it was closed. So the total energy in the last five minutes is less as a sum. Does that make sense?

```
|-----2-----2-----2-----2-----2-----2| Docker running, using 2 ei every second = 12
|-----2-----2-----2-----2------------| Docker killed, not using anything = 10
```

**Comment by ArneTR** [comment]
I was asking bc I am unsure if that is something you implemented or macOS does.

EnergyImpact is newer shown in macOS after an application closes, right? So the question is how this abatement is brought in. Is that your code?

I understand that if you add up *energy* values and at some point you close the program the energy in the sliding window goes down. So I understand if EnergyImpact does behave the same. I was under the impression that it is more like a power value and thus I would not have expected it to go down.

So if you have implemented that abatement yourself: How do you reduce it. Linearly?
